src/roles.js

```diff
@@ -344,8 +377,8 @@ export class Capabilities {
 /**
  *
  * @param {string} roleId
- * @returns {roleId is keyof DEFAULT_CAPABILITIES}
+ * @returns {roleId is keyof ROLES}
  */
 function isKnownRoleId(roleId) {
-  return roleId in DEFAULT_CAPABILITIES
+  return roleId in ROLES
```

Comment:
Highlighting this: `DEFAULT_CAPABILITIES` didn't have the creator or "no role" capabilities before. `ROLES` _does_ have those. I _think_ this is fine based on usage but want to make sure.

Reply:
I think we should avoid allowing the creator role id, maybe not in this function but instead where it is used on line 288? We will later validate role assignments, but I think it's safer that we don't allow a role record to define a role to be the project creator, since we're not allowing the project creator role to be assigned, just implicit.

Reply:
Done in f24c85e3d807ce58e3f4b99a148f6ee1c9d345b6.